packages/terra-clinical-data-grid/src/subcomponents/SectionHeader.jsx

```diff
@@ -113,7 +113,8 @@ class SectionHeader extends React.Component {
     return (
       <div
         className={cx('section-header')}
-        data-section-header
```

Comment:
I would leave this one the way it is in the interest of passivity. Might be a little awkward but we can revisit it in the future if we need to (or just remove it outright if it's not being used).

Reply:
I didn't see anyone using it which is why I changed it.

Reply:
We're not using it internally, but a theoretical consumer might be using it for some other purpose, and changing it would be non-passive for them.